Sumo-Logic-Tools/Collector_Management/Alert_Collector_not_sending_data_within_X_hours.txt

```diff
@@ -9,7 +9,8 @@ _index=sumologic_volume _sourceCategory="collector_volume"
 | formatDate(fromMillis(most_recent),"yyyy/MM/dd HH:mm:ss") as most_recent_time 
 | toMillis(now()) as currentTime 
 | formatDate(fromMillis(currentTime),"yyyy/MM/dd HH:mm:ss") as search_time 
-| where (currentTime-most_recent) > 18000000 // Threshold setting is milliseconds - currently set to 5 hours
+//| where (currentTime-most_recent) > 18000000 // Threshold setting is milliseconds - currently set to 5 hours
+| where (currentTime-most_recent) > 604800000 // Threshold setting is milliseconds - currently set to 7 days
 | fields most_recent_time,total_in_mb,collector
 | format ("%s has not sent data for at least 5 hours", collector) as collector_status // The "5 hours" is equivalent to the previous Threshold setting in milliseconds
```

Comment:
I see you changed this from 5h to 7d?
1) Can we remove/update the comment in the query to account for the change?
2) Additionally, Why 7 days instead of 5 hours? (what is the reasoning for the amount of time chosen?)
3) Maybe that time should be a param for the script?

Reply:
Ah, yes please, I should have done that before my commit.

Reply:
It's not too late :) When you've made the changes, simple commit again, and the PR will be automagically updated ;)

Reply:
Also, I fired up a fresh Win 10 box to test this one and am seeing the following. 
http://sumologic.link/e4b9c2b8be8c
Are you seeing this also, or am I just too out of sync w/ my windows-foo to realize there is an easy fix for this?

Reply:
Yes, I see it too, but it's a Python thing on Windows.... 

You can ignore it or do this to quiesce the warning verbosity: 

python -W ignore delete_ghost_collectors.py